topics/diffpriv/index.md

```diff
@@ -0,0 +1,10 @@
+---
+aliases: differential-privacy, privacy
+display_name: DiffPriv
+short_description: Differential Priacy, or diff-priv, is a way to display trends in a dataset while keeping an indiviuals data private.
```

Comment:
2 things:

1) You wrote "Priacy", not "privacy".
2) Differential Privacy doesn't only display trends. It can display other general statistics while keeping a users data private.